### DISCUSSION TITLE
[Feature] Open modal when user clicks on airport or route

### DISCUSSION THREAD

**Discussion by edbourque0** [johanohly/AirTrail#153] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I think it would be great to have a modal open up when a user clicks on an airport or a route on the map to have more information about them. For the airport view we could see some stats like number of flights to and from this airport, average flight time to/from this airport and have a view of all the flight since we can't see this right now (it only shows XX+ flights). For the routes, I would like to  have a modal that only shows this route on a map and have some stats about it.

**Comment by johanohly** [comment]
Thank you for making this feature request! This sounds like a pretty nice addition, but it does collide with your other feature request #154, as you suggested on them both, that they trigger by clicking on a route / airport.

**Reply by edbourque0** [reply]
Got it, sorry about the duplicate features.

**Reply by johanohly** [reply]
No worries :)
I am wondering which of the two you would prefer? 
I could also imagine greying out any unrelated routes when hovering instead, and implement the modal on click...

**Reply by edbourque0** [reply]
Yeah that's a great idea, I think the grey on hover + modal on click combo would be a great hybrid between the two!

**Comment by johanohly** [comment]
This was added in v3.2.0
